### ISSUE TITLE
Black frames insertion for reversed videos

### ISSUE THREAD

**Issue by SuslikV** [OpenShot/libopenshot#541]
**Description**
When `openshot::Settings::Instance()->WAIT_FOR_VIDEO_PROCESSING_TASK` is set to _true_ (in OpenShot the _Process Video Frames in Parallel (Experimental)_ is unchecked) reversed video at 1x speed has black frames inserted.

**Details**
The `GetImage()` call from the:

https://github.com/OpenShot/libopenshot/blob/4ed575f1bba089dba808d50a15675ea27199ccd8/src/Clip.cpp#L350

produces solid black frames (at full resolution of the project, not the Preview's size by the way) that is put into the cache and displayed later until the cache invalidates (Preview window resized for example).

There is no pattern in the black frames insertion. I happens more frequently if debug is enabled but it is still random.

**Steps to reproduce**
1. Luck, You need it. ( maybe the https://github.com/OpenShot/openshot-qt/issues/2833 https://github.com/OpenShot/openshot-qt/issues/3451 )
2.  Project file for OpenShot v 2.5.1 to reproduce the issue. You may take this one:

[Test backward 1x black frames insertion.zip](https://github.com/OpenShot/libopenshot/files/4891207/Test.backward.1x.black.frames.insertion.zip)

about 174KB to download CRC of the `.zip` archive MD5:08C6787B042B42B1D31B4834E9C9E948

3. Either try to export the file and play it back or make the  _Process Video Frames in Parallel (Experimental)_ unchecked in _Preferences_ of OpenShot and just playback the project.

The test consist of single image movie 25 fps reversed at 1x and cut to 4 sec. It has white text on the black background. It is about 4 sec long (cut). It flickers when blank image is inserted during export (or even during playback if "process frames in parallel" is unchecked).

**Why it is important?**
Because in the current build of the `openshot-qt` during the _Export_ the mentioned above flag forced to `True`:
https://github.com/OpenShot/openshot-qt/blob/cb81d4c1631ad44ceca32bb71233c6e346b4958b/src/windows/export.py#L118
This makes impossible to export files without damage that has some fragments of video reversed during OpenShot edits.

**Side notes**
The way the audio is reversed is out of scope for this issue. This is different thing.

**Comment by yaw-man** [comment]
This issue was not fixed. I just encountered it today. Can it be re-opened?
